src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/textUnit.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { SERVER_API_URL } from 'app/app.constants';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { TextUnit } from 'app/entities/lecture-unit/textUnit.model';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { LectureUnitService } from 'app/lecture/lecture-unit/lecture-unit-management/lectureUnit.service';
+
+type EntityResponseType = HttpResponse<TextUnit>;
+type EntityArrayResponseType = HttpResponse<TextUnit[]>;
```

Comment:
GitHub Actions complains: `'EntityArrayResponseType' is defined but never used`

Reply:
Done in e881a30d6467f0e45c64b9490f1b0b6a4fef22f0